aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -179,6 +179,7 @@ func resourceAviatrixTransitGateway() *schema.Resource {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Default:     false,
```

Comment:
Should we change default to true so that people who disabled active mesh would see the warning? I think only people setting it to True would see the warning now.

Reply:
The logic is on 6.5.b, all non-ActiveMesh features are still supported, they can be created and modified, but they can't be upgraded to 6.6 since backend is blocking it. Since 6.6, all non-ActiveMesh features will be gone. If we change default from false to true, it surely will remind customers who did not set "enable_active_mesh" in their 
 non-ActiveMesh spokes/transits configuration files(if they set it to true or false, they will get the deprecation notice), however, those customers will get reached out anyway by our support.

Reply:
Sounds good